README.md

```diff
@@ -1,2 +1,57 @@
-# test-configs
-gsoc 24 project where will store configs for diff.groovy , https://github.com/orgs/checkstyle/projects/8
+# Checkstyle Test Configurations Generator
```

Comment:
Please put this content to contribution.md

Example https://github.com/checkstyle/checkstyle/blob/master/.github/CONTRIBUTING.md

But in Readme put details on how to use configs. 99% of usage of this repository will be by people who has 0 interest in java code of this repository. All user care is just how to use configs of this repository.

Reply:
Still not done

Reply:
Is the README right? I am preparing for CONTRIBUTION.md.